docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -424,13 +429,14 @@ The Photos layer renders a photo at each data point.
 The Network layer is currently in [public preview](/docs/release-life-cycle/). Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.
 {{% /admonition %}}
 
-The Network layer renders a network graph. This layer supports the same [data format supported by the node graph visualization](ref:data-format-supported-by-the-node-graph-visualization) with the addition of [geospatial data]({{< relref "#location">}}) included in the nodes data. The geospatial data is used to locate and render the nodes on the map.
+The Network layer renders a network graph. This layer supports the same [data format supported by the node graph visualization](ref:data-format-supported-by-the-node-graph-visualization) with the addition of [geospatial data]({{< relref "#location-mode">}}) included in the nodes data. The geospatial data is used to locate and render the nodes on the map.
```

Comment:
What's the page that you're linking to in the first link?

Reply:
Yes I created that one last week :)

Reply:
Is this the page and heading you're referring to? I have to add the link in the front matter of the page for the reference to work:
https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/node-graph/#data-requirements

Reply:
Oh sorry, I didn't see it when I looked in the front matter.